translations/de.ts

```diff
@@ -2205,26 +2210,26 @@ Viele Grüße
         <location filename="../src/net/toxdns.cpp" line="103"/>
         <source>Unexpected number of values in text record</source>
         <comment>Error with the DNS</comment>
-        <translation>Unerwartete Anzahl an Einträgen in Text</translation>
+        <translation>Unerwartete Anzahl an Einträgen im Texteintrag</translation>
     </message>
     <message>
         <location filename="../src/net/toxdns.cpp" line="132"/>
         <source>The version of Tox DNS used by this server is not supported</source>
         <comment>Error with the DNS</comment>
-        <translation>Die von diesem Server verwendete Version von Tox DNS wird leider nicht unterstützt</translation>
+        <translation>Die von diesem Server verwendete Version von Tox-DNS wird leider nicht unterstützt</translation>
     </message>
     <message>
         <location filename="../src/net/toxdns.cpp" line="144"/>
         <source>The DNS lookup does not contain any Tox ID</source>
         <comment>Error with the DNS</comment>
-        <translation>Die ToxDNS Nachfrage konnte keine Tox ID finden</translation>
+        <translation>Die ToxDNS-Nachfrage konnte keine Tox ID finden</translation>
```

Comment:
original sagt "The DNS lookup" aber es handelt sich um ToxDNS, was wohl nicht übersehtzt werden soll. Der ursprüngliche Übersetzer hat dann lookup mit Nachfrage übersetzt. Kann man aber imho ganz weglassen.